src/components/Header.tsx

```diff
@@ -37,14 +37,14 @@ export default function Header() {
           <a href="/#">Help</a>
           <a href="/#">Find In-Store</a>
           {headerIcons.map((image, index) => (
-            <span className="flex justify-items-center">
+            <span className="flex justify-items-center" key={index}
```

Comment:
justify-items-center seems to be incorrect here

Reply:
addressed. Added Items-center and added space-between in parent div to have them approach on both ends.